### ISSUE TITLE
Message failed to import. - (Description length must be less than or equal to 2048. (Parameter 'Description'))

### ISSUE THREAD

**Issue by SebasRebazCoding** [Ethorbit/Discord-Chat-Importer#11]
Hello there,
I'm trying to use the Discord importer to rescue the content some friends and I have, several memories and development details alike. But some of the messages sent are bigger than the character limit of 2048 that descriptions have. I am aware that there is a "split" command or similar, but lack the knowledge to advance any further. Would it be possible to update the way in which messages are handled so it can split the longer messages?

**Comment by WhakEi** [comment]
Yeah @Ethorbit, if there's a way to make it skip or split a 2048 char message it would be awesome.
Thanks in advance

**Comment by Emu-king** [comment]
i Fixed this problem for me

I went into the file using notepad++  found the big text that is more than 2000 characters which should be the next message before the error message appeared.
I edited the text to shorten it within the Quotation marks
then i deleted everything before the text message until i hit the "messages": [ 
 because i did not want the bot to redo everything again then i started it back up and it continued where it left off.
 
 i hope this helps
 @WhakEi @SebasRebazCoding
